src/backend/api/Fusion.Resources.Api/Controllers/Personnel/ApiModels/ApiInternalPersonnelPerson.cs

```diff
@@ -139,14 +139,14 @@ public PersonnelAbsence(QueryPersonAbsenceBasic absence)
             {
                 Id = absence.Id;
                 AppliesFrom = absence.AppliesFrom.UtcDateTime;
-                AppliesTo = absence.AppliesTo is null ? absence.AppliesFrom.UtcDateTime : absence.AppliesTo!.Value.UtcDateTime;
```

Comment:
hehe dunno who did this or did CR on it.. must be alcohol in the picture or something 🙈 🍺